.gitignore

```diff
@@ -4,3 +4,16 @@
 
 /EventManager.sln
 /.vs/EventManager/v14
+/Web.config
+/.vs
+/node_modules
+/server/typings
+/Server/dist
```

Comment:
Add client/dist to this list

Reply:
My server actually serves the index.html + bundle.js from that folder, removing it would make the client inaccessible. Is there a better way to render my react app from node.js server?

Reply:
Try html-webpack-plugin.
These links should be helpful: https://webpack.js.org/plugins/html-webpack-plugin/
https://www.codecademy.com/articles/react-setup-iv